docs/installation.md

```diff
@@ -218,19 +218,174 @@ On Linux, as an optional post-installation task, you may wish to setup the bot t
 
 ------
 
-### Anaconda
+## Installation with Conda (Miniconda or Anaconda)
 
-Freqtrade can also be installed using Anaconda (or Miniconda).
+Freqtrade can also be installed with Miniconda or Anaconda. Conda (Miniconda or Anaconda) would automatically prepare and manage the extensive library-dependencies of the Freqtrade program.
 
-!!! Note
-    This requires the [ta-lib](#1-install-ta-lib) C-library to be installed first. See below.
+##### What is Conda?
+
+It is: (1) package, (2) dependency and (3) environment management for any programming language : https://docs.conda.io/projects/conda/en/latest/index.html
+
+Shortly : Conda < Miniconda < Anaconda. Check : https://linuxnetmag.com/miniconda-vs-anaconda/
+
+It is recommended to install Miniconda, not Anaconda. The difference between both lies in the amount of packages that would be installed upfront. Difference in weight is around like 1:8 : https://www.dunderdata.com/blog/anaconda-is-bloated-set-up-a-lean-robust-data-science-environment-with-miniconda-and-conda-forge
+
+
+#### 1. Instal Conda
+
+3 step installation instruction can be found here :
+https://conda.io/projects/conda/en/latest/user-guide/install/linux.html#install-linux-silent
+
+Confirm with `yes` all questions. After installation, it is mandatory to turn your terminal OFF and ON again.
```

Comment:
i'd be careful with recommending "say yes to everything"  about something we don't control.

it may be the correct answer now - but we don't know how the installer changes in the future, nor do we know the users preferences - so i think it's best to be neutral on this - with "answer all questions" or something similar.

Reply:
OK